js/IceStorm/weather/README.md

```diff
@@ -13,28 +13,35 @@ flowchart LR
     icestorm --report--> s3[Station #3]
 ```
 
-Follow these steps to build and run the demo:
+## Ice prerequisites
 
-1. Install the dependencies:
+Install IceStorm. See [Ice service installation].
```

Comment:
Should we include IceStorm config with this demo and the instructions to start IceStorm?

Reply:
No, I would leave this to the server-side of this demo.

Then, the question is should we include IceStorm (IceGrid, Glacier2) as Ice prerequisites of these client-side-only demos? Maybe not.

Reply:
We should remove the Glacier2 config, and rework its README to be consistent.